tools/physiofit/physiofit.xml

```diff
@@ -52,29 +69,35 @@
                 <option value='json'>config file (json)</option>
             </param>
             <when value='tsv'>
-                <param name='datafile' type='data' format='tsv, txt' multiple='true' label='Upload data file containing biomass and concentration values'/>
+                <param name='datafile' type='data' format='tabular, txt' multiple='true' label='Upload data file containing biomass and concentration values'/>
             </when>
             <when value='json'>
-                <param name='datafile' type='data' format='tsv, txt' multiple='true' label='Upload data file containing biomass and concentration values'/>
+                <param name='datafile' type='data' format='tabular, txt' multiple='true' label='Upload data file containing biomass and concentration values'/>
                 <param name='cfgfile' type='data' format='json' label='Upload configuration file' />
             </when>
         </conditional>
 
 
         <section name='basic_settings' title='Basic Settings'>
 
-            <param name='lag' type='boolean' argument='--lag' label='Flag to estimate lag phase' truevalue='-l' falsevalue=''/>
+            <param name='lag' type='boolean' label='Flag to estimate lag phase'/>
```

Comment:
The previous version looks better. Using argument is also encouraged :-)

Reply:
True, I don't know why I changed that ^^